controllers/userController.js

```diff
@@ -12,6 +13,7 @@ module.exports = {
     try {
       let apps = await GreenPickApp.find({ userId: req.user._id });
       req.data = apps;
+      res.locals.apps = apps;
```

Comment:
it's best to only use req.data or res.locals.apps, not both

Reply:
> API logic should be in the API folder, so please move the functions to a new controller^^ just like categoryController

Yeah i did it at first actually, but then inside there'll be only `respondJSON` function. i wouldn't duplicate the `getAllApps `code from `userController`.. so i wasn't sure if it's worth to create a new file. 

Idk how @DariaBezuhla and @simbuzan  going to do the favorites.. are you having the getAllFavApps in userController and probably the respondJSON in another new api controller file?

Reply:
you can copy the getAllApps from userController into your new file. As soon as we have Vue for that part, we can delete it from userController (since it won't be used there)